src/main/java/org/gridsuite/modification/server/entities/ModificationEntity.java

```diff
@@ -49,8 +49,8 @@ public class ModificationEntity {
     @Setter
     private ModificationGroupEntity group;
 
-    @Column(name = "stashed")
-    private Boolean stashed;
+    @Column(name = "stashed", columnDefinition = "boolean default false")
+    private Boolean stashed = false;
```

Comment:
is it nullable?
```suggestion
    @Column(name = "stashed", columnDefinition = "boolean default false", nullable = false)
    private boolean stashed = false;
```

Reply:
will change the database schema, I didn't want to in this PR.